tests/task_queue_stats_test.go

```diff
@@ -41,23 +57,23 @@ type (
 	TaskQueueExpectationsByType map[enumspb.TaskQueueType]TaskQueueExpectations
 )
 
-// TestTaskQueueStatsSuite tests querying task queue stats.
-//
-// There are currently three ways to do that:
-// 1. DescribeTaskQueue with ReportStats=true
-// 2. DescribeTaskQueue with ApiMode=ENHANCED and ReportStats=true [deprecated]
-// 3. DescribeWorkerDeploymentVersion with ReportTaskQueueStats=true
-//
-// Unless a test calls out a specific methods, all three methods are tested in each test case.
-func TestTaskQueueStatsSuite(t *testing.T) {
+// TODO(pri): remove once the classic matcher is removed
+func TestTaskQueueStatsSuite_ClassicMatcher(t *testing.T) {
```

Comment:
the naming scheme for the others is:

```suggestion
func TestTaskQueueStats_Classic_Suite(t *testing.T) {
```

I don't care that much but just keep them easy to type and consistent